frontend/src/pages/account/Create.vue

```diff
@@ -171,49 +174,49 @@ export default {
          */
         validateFormInputs () {
             if (!this.input.username.trim()) {
-                this.errors.username = 'Username is required'
+                this.errors.username = this.$t('auth.errors.usernameRequired')
```

Comment:
In the same way we have the fallbacks in place for the HTML, should we not also have them here, e.g.:

```js
this.errors.username = this.$t('auth.errors.usernameRequired') || 'username is required'
```